torch/_inductor/cpu_vec_isa.py

```diff
@@ -228,6 +228,27 @@ def __str__(self) -> str:
   _tile_dpbf16ps(0, 1, 2);
   _tile_dpbusd(0, 1, 2);
 }
+"""
+
+    _amx_fp16_code = """
+#include <cstdint>
+#include <immintrin.h>
+
+struct amx_tilecfg {
+  uint8_t palette_id;
+  uint8_t start_row;
+  uint8_t reserved_0[14];
+  uint16_t colsb[16];
+  uint8_t rows[16];
+};
+
+extern "C" void __amx_chk_kernel() {
+  amx_tilecfg cfg = {0};
+  _tile_loadconfig(&cfg);
+  _tile_zero(0);
+  _tile_dpfp16ps(0, 1, 2);
+  _tile_dpbusd(0, 1, 2);
+}
```

Comment:
Can we reduce the duplicated with `_amx_code`?

Reply:
Revised.